tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -54,6 +54,7 @@ def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os):
         "centos7": "defaults,_netdev,flock,user_xattr,noatime,noauto,x-systemd.automount,"
         "x-systemd.requires=lnet.service",
         "alinux": "defaults,_netdev,flock,user_xattr,noatime",
+        "alinux2": "defaults,_netdev,flock,user_xattr,noatime",  # TODO: ensure this is correct
```

Comment:
Has this been tested now?

Reply:
No. I've only run a subset of the integration tests.

Reply:
@tilne Amazon Linux 2 is systemd based so it has the full set of options in `default`. Rebase and you'll see you don't need this line anymore